src/js/pensions/createApplicationStatus.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default function createPensionApplicationStatus(store) {
+  const root = document.getElementById('react-pensionApplicationStatus');
+
+  if (root) {
+    import(
+      /* webpackChunkName: "pension-application-status" */
+      '../common/schemaform/ApplicationStatus').then(module => {
+      const ApplicationStatus = module.default;
+      ReactDOM.render((
+        <Provider store={store}>
+          <ApplicationStatus
+            formId="21P-527EZ"
+            titleText="Pension application"
```

Comment:
That's a good question. I did that initially, but it seemed kind of long. @stacyhlinn or @peggygannon?

Reply:
@jbalboni @cehsu I think we should include the full form title, but it would be helpful if I could see a screenshot of how it would look on this page before moving forward with that decision for all the forms.

Reply:
@peggygannon 

![screen shot 2017-09-18 at 12 55 47 pm](https://user-images.githubusercontent.com/634932/30553941-c2193c7e-9c70-11e7-8a18-e3b98de03ff3.png)

Reply:
Thanks, @jbalboni. I do think we should include the full form name for this message. This will be especially helpful for EDU, since there are several education forms.